src/kibana/components/vislib/lib/validate_visdata.js

```diff
@@ -0,0 +1,70 @@
+define(function (require) {
+  var _ = require('lodash');
+
+  return function ValidateVisDataFactory() {
+
+    /**
+     * Prevents missing data and null values from being passed to the vislib.
+     * Missing data occurs when an array of data values is empty.
+     * @param data {Object} Kibana data object
+     * @returns {ValidateVisData}
+     * @constructor
+     */
+    function ValidateVisData(data, chartType) {
```

Comment:
Is there a reason this is a class instead of just a function? It doesn't seem to utilize any of the features of classes, and the class instance isn't kept around: `var validData = new ValidateData(data, chartType).visData;`